app/controllers/health_cards_controller.rb

```diff
@@ -7,7 +7,34 @@ class HealthCardsController < ApplicationController
 
   def show
     respond_to do |format|
-      format.healthcard { render json: { verifiableCredential: [jws.to_s] } }
+      vc = jws.to_s
+      format.healthcard { render json: { verifiableCredential: [vc] } }
+      format.fhir_json do
+        fhir_params = FHIR.from_contents(request.raw_post)
```

Comment:
For this code, I'd suggest we add an `app/lib` directory, and stick classes in there to handle this. So this becomes something along the lines of:
```
format.healthcard { render json: HealthCardsFileIssuer.new(some_params).issue }
format.fhir_json { render json: HealthCardsFHIRIssuer.new(some_other_params).issue }
```

Reply:
I think we can move a lot of the rails specific code to something in `app/lib` but I think a good portion of this functionality is non-rails specific and could benefit implementers if it's in the library. @radamson - Thoughts?

Reply:
To recap our discussion with @Jammjammjamm I think that.

- The FHIR Request Parameter checks should stay out of the `health_cards` library, but it could be moved into `lib` so that logic can stay out of the controller.
- Create an `Exporter` module in the `health_cards` library that takes in a jws and returns in in the desired export format (e.g. in a parameters resource, wrapped in json for a file download)